Source/player.cpp

```diff
@@ -3464,9 +3464,15 @@ void PlayDungMsgs()
 		myPlayer.Say(HeroSpeech::IMustBeGettingClose, 40);
 		myPlayer.pDungMsgs |= DungMsgHell;
 	} else if (!setlevel && currlevel == 16 && !myPlayer._pLvlVisited[16] && (myPlayer.pDungMsgs & DungMsgDiablo) == 0) {
-		sfxdelay = 40;
-		sfxdnum = SfxID::DiabloGreeting;
-		myPlayer.pDungMsgs |= DungMsgDiablo;
+		for (auto &monster : Monsters) {

```

Comment:
Technically we prefer not having auto here since the type isn't immediately obvious. But heck I don't remember what the monster struct name is atm.